awx/settings/development_defaults.py

```diff
@@ -0,0 +1,129 @@
+# MIDDLEWARE = "@insert debug_toolbar.middleware.DebugToolbarMiddleware"
+
+DEBUG_TOOLBAR_CONFIG = {'ENABLE_STACKTRACES': True}
+
+# Configure a default UUID for development only.
+SYSTEM_UUID = '00000000-0000-0000-0000-000000000000'
+INSTALL_UUID = '00000000-0000-0000-0000-000000000000'
+
+# Ansible base virtualenv paths and enablement
+# only used for deprecated fields and management commands for them
+BASE_VENV_PATH = os.path.realpath("/var/lib/awx/venv")
+
+CLUSTER_HOST_ID = socket.gethostname()
+
+AWX_CALLBACK_PROFILE = True
+
+# ======================!!!!!!! FOR DEVELOPMENT ONLY !!!!!!!=================================
+# Disable normal scheduled/triggered task managers (DependencyManager, TaskManager, WorkflowManager).
+# Allows user to trigger task managers directly for debugging and profiling purposes.
+# Only works in combination with settings.SETTINGS_MODULE == 'awx.settings.development'
+AWX_DISABLE_TASK_MANAGERS = False
+
+# Needed for launching runserver in debug mode
+# ======================!!!!!!! FOR DEVELOPMENT ONLY !!!!!!!=================================
+
+# Store a snapshot of default settings at this point before loading any
+# customizable config files.
+# this_module = sys.modules[__name__]
+# local_vars = dir(this_module)
+# DEFAULTS_SNAPSHOT = {}  # define after we save local_vars so we do not snapshot the snapshot
+# for setting in local_vars:
+#     if setting.isupper():
+#         DEFAULTS_SNAPSHOT[setting] = copy.deepcopy(getattr(this_module, setting))
+
+# del local_vars  # avoid temporary variables from showing up in dir(settings)
+# del this_module
+#
+###############################################################################################
+#
+#  Any settings defined after this point will be marked as as a read_only database setting
+#
+################################################################################################
+
+# If there is an `/etc/tower/settings.py`, include it.
+# If there is a `/etc/tower/conf.d/*.py`, include them.
+# include(optional('/etc/tower/settings.py'), scope=locals())
+# include(optional('/etc/tower/conf.d/*.py'), scope=locals())
+
+# If any local_*.py files are present in awx/settings/, use them to override
+# default settings for development.  If not present, we can still run using
+# only the defaults.
+# this needs to stay at the bottom of this file
+# try:
+#     if os.getenv('AWX_KUBE_DEVEL', False):
+#         include(optional('development_kube.py'), scope=locals())
+#     else:
+#         include(optional('local_*.py'), scope=locals())
+# except ImportError:
+#     traceback.print_exc()
+#     sys.exit(1)
+
+
+# The below runs AFTER all of the custom settings are imported
+# because conf.d files will define DATABASES and this should modify that
+# from .application_name import set_application_name
+
+# set_application_name(settings.DATABASES, CLUSTER_HOST_ID)  # NOQA
+
+# del set_application_name
+
+# settings.populate_obj(sys.modules[__name__])
```

Comment:
Is there a replacement for the commented code here? Is it possible to obtain these behaviors (like `local_*.py`) with dynaconf?

Reply:
https://github.com/ansible/awx/pull/15702/files#diff-7c44e0fb5f02a0b7ba5fc1d0e6b57da7b35edf564011c78e45d356343b8595b6R16-R33